apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/logging/api/ILog.java

```diff
@@ -51,5 +51,10 @@
 
     void debug(String format, Object... arguments);
 
+    default void debug(Throwable t, String format, Object... arguments) {
```

Comment:
this `default` for?

Reply:
> this `default` for?

There’re many anonymous inner classes implementing this interface in unit test, I just don’t want to implement this method there over and over, it’s useless

Reply:
This is not a good start for the codes. Others may refer to this in further PRs.